frontend/templates/product-list/SecondaryNavigation.tsx

```diff
@@ -19,6 +21,15 @@ export function SecondaryNavigation({ productList }: SecondaryNavigationProps) {
       productList.type === ProductListType.ToolsCategory;
    const hasDeviceNavigation =
       productList.type !== ProductListType.AllParts && !isToolsProductList;
+   const { currentItemTitle, ancestors } = useProductListAncestors(productList);
+   const breadCrumbs: BreadcrumbItem[] = ancestors.map((ancestor) => ({
+      label: ancestor.title,
+      url: productListPath(ancestor),
+   }));
+   breadCrumbs.push({
+      label: currentItemTitle,
+      url: undefined,
```

Comment:
This adds clarity but is unneeded. Up to you if you leave it.

```suggestion
```

Reply:
We also could make this link the to current page or generate the correct link for the current page?